src/main/java/core/basesyntax/Engine.java

```diff
@@ -1,5 +1,76 @@
 package core.basesyntax;
 
-public class Engine {
+public class Engine implements Cloneable {
```

Comment:
btw, this is requierd if you use super() in clone method, because otherwise it will generate CloneNotSupportedException. In your case you used constructor, so interface will not influence the code execution. But it's not critical, you can leave it here